.flowconfig

```diff
@@ -1,6 +1,7 @@
 [ignore]
 .*/node_modules/react-native/.*
 .*/node_modules/fbjs/.*
+node_modules/config-chain/test/broken.json$
```

Comment:
I also cannot reproduce nor find that file locally so I need to go a bit blind here.